src/components/LeftNav.tsx

```diff
@@ -432,7 +437,7 @@ const SubHeading = ({
   Icon: React.ReactComponentElement<any>;
 }) => {
   return (
-    <NavLink to={path} style={subheadingStyle} activeStyle={activeStyle} exact>
+    <NavLink end to={path} style={subheadingStyle} activestyle={activestyle}>
```

Comment:
Ahhh I see, the `activestyle` is the HTML attribute. Makes sense. Still, please change the JavaScript object's name to `camelCase`

Reply:
Ok well now you know what I meant and I explained my thoughts above :)